pages/linux/qmmonitor.md

```diff
@@ -0,0 +1,8 @@
+# qm monitor<vmid>
+
+> To Enter a Qemu Monitor Interface.
```

Comment:
```suggestion
> Enter to Qemu Monitor Interfaces.
```

Reply:
Thank you for pointing out my errors.I am new to GitHub and it is very helpful.

Reply:
Remember that you can use this GitHub feature <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request>.
It's the easier way to apply suggestions in `tldr`

Reply:
Thank you very much❤️